libs/map/src/components/legend/vl-map-legend.cy.ts

```diff
@@ -295,7 +299,9 @@ describe('component vl-map-legend - wfs layer', () => {
     it('should contain the correct style item with an icon and name', () => {
```

Comment:
dit gaat eigenlijk enkel controleren of er 1 style item is, niet of de correcte style item er in zit;
ik stel voor; 

- of verander beschrijving test; bv. `should contain one style item with an icon and a name`
- of als het kan zorg, dat de test zelf de beschrijving matcht

mogelijk begrijp ik dit verkeerd; maar de dezelfde test voor niet-wfs en wfs gaat slagen, onafhankelijk over welke het gaat?

Reply:
Ik heb de test aangepast om de style te controleren.
Voor wms lagen gaat deze test niet slagen. Een wms style item heeft geen icon en name. Alleen een image waarin beide afgebeeld zijn.